File: NeighbourlyServer/src/UserInfoMessage.java
import java.util.ArrayList;

public class UserInfoMessage extends Message {
	int userID;
	String name;
	String email;
	String imageURL;
	ArrayList<Item> myItems;
	ArrayList<Item> borrowedItems;
	
	UserInfoMessage(int userID,String email, Database database )
	{
		super("valid");
		this.userID = userID;
		this.name=database.getNameFromID(userID);
		this.email = email;
		this.imageURL = database.getUserImageURLfromUserID(userID);
		this.myItems = database.getMyItems(userID);
		this.borrowedItems = database.getBorrowedItems(userID);
	}
	
	UserInfoMessage(int userID, Database database)
	{
		super("valid");
		this.userID = userID;
		this.name=database.getNameFromID(userID);
		this.myItems = database.getMyItems(userID);
		this.borrowedItems = database.getBorrowedItems(userID);
		this.imageURL = database.getUserImageURLfromUserID(userID);
	}

}

File: NeighborlyWorkSpaceV2/Neighborly/MessageClasses/backToFront/UserInfoMessage.swift
//
//  UserInfoMessage.swift
//  Neighborly
//
//  Created by Other users on 4/14/18.
//  Copyright © 2018 Adam Liber. All rights reserved.
//

import Foundation

struct UserInfoMessage:Codable{
    
    var userID:NSInteger?
    
    var name:String?
    
    var email:String?
    var imageURL:String?
    var myItems:[Item]?
    var borrowedItems:[Item]?
    var message:String
    
}
